target/linux/ath79/dts/qca955x_senao_loader.dtsi

```diff
@@ -34,6 +34,15 @@
 	};
 };
 
+&pcie1 {
+	status = "okay";
```

Comment:
This will activate the `pcie1` node on all devices using this dtsi file.
Is this intended?

Reply:
yeah I think it's harmless, OEM has both root bus enabled too

unless you prefer leaving out "status = okay" from the DTSI and put them in device DTS for only the one pcie bus that has hardware on it

Reply:
Yes I would prefer if you set the `status = okay` only for the boards where it is actually used.

Reply:
sorry for the wait, I addressed the addition of pcie1 by adding `status` to individual DTS for both pcie in a separate commit